### DISCUSSION TITLE
[Windows 11 Notification Center Styler] [Windows 11 Start Menu Styler] Translucent/Transparent backrgorund mods on a white window

### DISCUSSION THREAD

**Discussion by Better-Than-You** [ramensoftware/windhawk-mods#1041] [Q&A]
Well, you see Im using the translucent start menu, and notification mod. It looks awesome. But, when opening the start menu or the notification bar with a white window in background, I can't see anything at all. The window text color is white, and the background being transparent, the white from the back and the text gets mixed up.

Any work arounds?

**Comment by m417z** [comment]
It was discussed here:
https://github.com/ramensoftware/windows-11-start-menu-styling-guide/issues/76

**Reply by Better-Than-You** [reply]
Oh, that discussion didnt pop up in my search, sorry.
